### DISCUSSION TITLE
IntelliSense [ESP_LOGI]** cannot be found in my VS Code

### DISCUSSION THREAD

**Discussion by Bader-EE** [Mair/esp32-course#125] [Course Qeustions] [answered]
Hello,

as you write in the ESP32 course
> Environment Setup section [2]
> Installing and configuring VS Code video [4]
> at the time: 9:20

the **IntelliSense [ESP_LOGI]** cannot be found in my VS Code. Also, the other IntelliSense as well [see attached figure].
 I would know if is it a problem? should I do something? or that is the default.

<img width="763" alt="IntelliSense ESP_LOGI" src="https://user-images.githubusercontent.com/108470079/177499159-27068a50-2eb2-41fd-b34c-3510927c1499.png">

**Comment by Mair** [comment, marked as answer]
your header is incorrect
#include "esplog.h>"

should be "esp_log.h"
